packaging/entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+WD=$( pwd )
+# Run alembic migrations first
+( cd /code/node_types && alembic upgrade heads ) || \
+( cd /code/plugins && alembic upgrade heads ) || \
+( cd /code/blueprints && alembic upgrade heads ) || \
+( echo "Alembic upgrade failed" && exit 1 )
```

Comment:
You'll need here:
```suggestion

 ! { ( cd /code/node_types && alembic upgrade heads ) && \
 ( cd /code/plugins && alembic upgrade heads ) && \
 ( cd /code/blueprints && alembic upgrade heads ) } && \
 ( echo "Alembic upgrade failed" && exit 1 )
```
So that all 3 run and in case of failure the failed alert will run

Reply:
No, I should have commented the code better, sorry.  The idea is to have a single `entrypoint.sh` which will work with any of the three containers we use.  Each of those is going to have only one of the `/code/{node_types,plugins,blueprints}` directories so the idea is to run alembic in at least one of them.  Or fail otherwise.

Reply:
Makes sense then